po/nl_NL.po

```diff
@@ -19,63 +19,39 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Poedit 2.2.3\n"
 
-#: src/config_rmw.c:70
-#, fuzzy
-#| msgid ""
-#| "\n"
-#| "# A folder can use the $UID variable.\n"
-#| "# WASTE=/mnt/fs/Trash-$UID\n"
+#: src/config_rmw.c:73
 msgid ""
 "\n"
 "# A folder can use the $UID variable.\n"
+"# WASTE=/mnt/fs/Trash-$UID\n"
```

Comment:
This is one of the variables I meant in my comment a minute ago. Not included in the translation anymore